examples/vae_simple.py

```diff
@@ -0,0 +1,322 @@
+import sys
+import os
+import numpy as np
+import theano
+import theano.tensor as T
+import lasagne as nn
+import time
+import pickle
+from PIL import Image
+from lasagne.layers import MergeLayer
+from theano.tensor.shared_randomstreams import RandomStreams
```

Comment:
We use another one in Lasagne (https://github.com/Lasagne/Lasagne/blob/6dd88f5fada20768087f29ae89cbd83980fe0a4e/lasagne/layers/noise.py#L6):
```python
from theano.sandbox.rng_mrg import MRG_RandomStreams as RandomStreams
```
I think that one should be faster, can you try?